WcaOnRails/app/controllers/results_controller.rb

```diff
@@ -150,8 +123,160 @@ def rankings
     end
   end
 
+  def records
+    support_old_links!
+
+    # Default params
+    params[:event_id] ||= "all events"
+    params[:region] ||= "world"
+    params[:years] ||= "all years"
+    params[:show] ||= "mixed"
+
+    @shows = ["mixed", "slim", "separate", "history", "mixed history"]
```

Comment:
If this is going to be a const variable I think you should make this a class and freezed variable!
ie:
```ruby
SHOW_TYPE=%(mixed slim separate history mixed\ history).freeze
```

somewhere at the beginning of the controller :)

Reply:
Even if it's only going to be used on one of the controller's actions? Should I also do this with `@quantities` and `@types`?